core/blockchain.go

```diff
@@ -85,7 +85,8 @@ var (
 	acceptedBlockGasUsedCounter  = metrics.NewRegisteredCounter("chain/block/gas/used/accepted", nil)
 	badBlockCounter              = metrics.NewRegisteredCounter("chain/block/bad/count", nil)
 
-	acceptedTxsCounter = metrics.NewRegisteredCounter("chain/txs/accepted/count", nil)
+	acceptedTxsCounter  = metrics.NewRegisteredCounter("chain/txs/accepted", nil)
+	processedTxsCounter = metrics.NewRegisteredCounter("chain/txs/processed", nil)
```

Comment:
nit: we used `inserts` for block should we use `insert` here as well?

Reply:
Hmm I followed the same way that we did this for the gas consumption here. Could go either way, but I'd prefer to leave this as is unless you feel strongly